modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetrics.macro

```diff
@@ -1894,5 +1894,12 @@ definition {
 			echo("timeout exceeded");
 		}
 	}
+	macro notViewNodeCircleIcon {
```

Comment:
Macro already merged, updated and renamed to `viewCurrentNodeCircleNotPresent`. Please change in your testcase